Index: artist.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html>\r\n<head>\r\n    <title>Liste des artistes</title>\r\n    <link rel=\"stylesheet\" type=\"text/css\" href=\"front-end/css/shakur.css\">\r\n</head>\r\n<body>\r\n<h1>Liste des artistes</h1>\r\n<ul>\r\n    {{range .}}\r\n        <li id=\"artist{{.ID}}\">\r\n            <img src=\"{{.ImageURL}}\" alt=\"{{.Name}}\" />\r\n            <h2>{{.Name}}</h2>\r\n            <p>Albums : {{.Albums}}</p>\r\n            <p>Pistes : {{.Tracks}}</p>\r\n            <p><a href=\"{{.Self}}\">Page de l'artiste</a></p>\r\n        </li>\r\n    {{end}}\r\n</ul>\r\n\r\n<script>\r\n    // Exemple de code JavaScript qui utilise les ID des artistes\r\n    var artist1 = document.getElementById(\"artist1\");\r\n    console.log(artist1);\r\n</script>\r\n</body>\r\n</html>\r\n
===================================================================
diff --git a/artist.html b/artist.html
--- a/artist.html	
+++ b/artist.html	
@@ -10,18 +10,14 @@
     {{range .}}
         <li id="artist{{.ID}}">
             <img src="{{.ImageURL}}" alt="{{.Name}}" />
-            <h2>{{.Name}}</h2>
-            <p>Albums : {{.Albums}}</p>
-            <p>Pistes : {{.Tracks}}</p>
-            <p><a href="{{.Self}}">Page de l'artiste</a></p>
+            <div class="info">
+                <h2>{{.Name}}</h2>
+                <p>Albums : {{.Albums}}</p>
+                <p>Pistes : {{.Tracks}}</p>
+                <p><a href="{{.Self}}">Page de l'artiste</a></p>
+            </div>
         </li>
     {{end}}
 </ul>
-
-<script>
-    // Exemple de code JavaScript qui utilise les ID des artistes
-    var artist1 = document.getElementById("artist1");
-    console.log(artist1);
-</script>
 </body>
 </html>
