Index: artist.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html>\r\n<head>\r\n    <title>Liste des artistes</title>\r\n    <link rel=\"stylesheet\" type=\"text/css\" href=\"front-end/css/shakur.css\">\r\n</head>\r\n<body>\r\n<h1>Liste des artistes</h1>\r\n<ul>\r\n    {{range .}}\r\n        <li id=\"artist{{.ID}}\">\r\n            <img src=\"{{.ImageURL}}\" alt=\"{{.Name}}\" />\r\n            <h2>{{.Name}}</h2>\r\n            <p>Albums : {{.Albums}}</p>\r\n            <p>Pistes : {{.Tracks}}</p>\r\n            <p><a href=\"{{.Self}}\">Page de l'artiste</a></p>\r\n        </li>\r\n    {{end}}\r\n</ul>\r\n\r\n<script>\r\n    // Exemple de code JavaScript qui utilise les ID des artistes\r\n    var artist1 = document.getElementById(\"artist1\");\r\n    console.log(artist1);\r\n</script>\r\n</body>\r\n</html>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/artist.html b/artist.html
--- a/artist.html	(revision 2d0fe62d072defc9c8342cb7ac192e51551cefa8)
+++ b/artist.html	(date 1677660585891)
@@ -17,11 +17,5 @@
         </li>
     {{end}}
 </ul>
-
-<script>
-    // Exemple de code JavaScript qui utilise les ID des artistes
-    var artist1 = document.getElementById("artist1");
-    console.log(artist1);
-</script>
 </body>
 </html>
